xline/src/server/xline_server.rs

```diff
@@ -51,6 +52,8 @@ pub struct XlineServer {
     lease_storage: Arc<LeaseStore>,
     /// Consensus client
     client: Arc<Client<Command>>,
+    /// Xline client
+    xline_client: Arc<Mutex<crate::client::Client>>,
```

Comment:
Is it necessary to put xlien_client in a Mutex? I see no mutable usage. Maybe `Arc<Client>` is enough?

Reply:
Is it a good idea to add this dependency to `client` here? I mean, the xline client is only used in `test` and `benchmark` before implementing the `Lock` feature.